src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -18,9 +18,18 @@ def load_arguments(self, _):
         c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the connected cluster.')
         c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
-        c.argument('https_proxy', options_list=['--https-proxy'], help='Https proxy url to be used.')
-        c.argument('http_proxy', options_list=['--http-proxy'], help='Http proxy url to be used.')
-        c.argument('no_proxy', options_list=['--no-proxy'], help='List of urls/CIDRs for which proxy should not to be used.')
+        c.argument('https_proxy', options_list=['--proxy-https'], help='Https proxy URL to be used.')
+        c.argument('http_proxy', options_list=['--proxy-http'], help='Http proxy URL to be used.')
+        c.argument('no_proxy', options_list=['--proxy-skip-destinations'], help='List of URLs/CIDRs for which proxy should not to be used.')
```

Comment:
This is a breaking change. Please evaulate the impact to customers. You may need to keep the old option and add deprecation message before finally removing it if necessary.

Reply:
the last version was not published, so there are no customers that would be using it

Reply:
https://github.com/Azure/azure-cli-extensions/pull/1878, this was the last PR in which the change was added. The version had not been upgraded in setup.py, so i guess it would not have been published right?